apps/etterlatte-trygdetid/src/main/kotlin/trygdetid/regler/BeregnTrygdetid.kt

```diff
@@ -265,31 +274,46 @@ val summerFaktiskNorge =
         }
     }
 
+val faktiskNorge =
+    RegelMeta(
+        gjelderFra = TRYGDETID_DATO,
+        beskrivelse = "Velg mellom beregnet faktisk nasjonal og overstyrt poengaar",
```

Comment:
Noen grunn til å bruke `aa` i stedet for `å` i beskrivelsene?

Reply:
Habit?